doc/it.xsemantics.doc/about.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<!--
```

Comment:
yes, `about.html` should not have a copyright header ( @mmews-n4 sorry, I had overlooked this in your PR this morning)